src/stamp/encoding/encoder/cobra.py

```diff
@@ -0,0 +1,57 @@
+from numpy import ndarray
+
+from stamp.encoding.config import EncoderName
+from stamp.encoding.encoder import Encoder
+from stamp.preprocessing.config import ExtractorName
+
+try:
+    import torch
+    from cobra.utils.load_cobra import get_cobra
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "cobra dependencies not installed."
+        " Please update your venv using `uv sync --extra cobra`"
+    ) from e
```

Comment:
Nice! Though the `import torch` statement should be outside the `try`/`except` block (it is already a dependency of `stamp`)